idd/Energy+.idd.in

```diff
@@ -74316,7 +74316,7 @@ ChillerHeaterPerformance:Electric:EIR,
 
 !*****************CONDENSING EQUIPMENT*********************
 CoolingTower:SingleSpeed,
-       \min-fields 16
+       \min-fields 28
```

Comment:
Increased the `Min-Fields` such the input processor fills in additional default values.